WcaOnRails/config/locales/pt-BR.yml

```diff
@@ -4360,9 +4366,67 @@ pt-BR:
       veja <a
       href="https://github.com/thewca/wca-regulations-translations#translation-instructions">as
       instruções no repositório GitHub</a>.
+  full_organizer_guidelines:
+    #original_hash: 5d602df
+    description: >-
+      Bem-vindo à coleção de Orientações para Organizadores do WCAC. Se você
```

Comment:
This was should be saying W**Q**AC instead of W**C**AC

Reply:
@pedrosino Would you mind re-submitting the update with the fix?